kiota_abstractions/serialization/untyped_list.py

```diff
@@ -0,0 +1,24 @@
+from .untyped_node import UntypedNode
+
+class UntypedList(UntypedNode):
```

Comment:
```suggestion
class UntypedArray(UntypedNode):
```
Is there any reason to not call it UntypedArray?

Reply:
Same as above, in Python we have `list`  instead of `Arrays`, but I can change it back to Array if you like?